txnsync/emulatorNode_test.go

```diff
@@ -315,10 +315,13 @@ func (n *emulatedNode) step() {
 			dm := peer.deferredSentMessages[0]
 			peer.deferredSentMessages = peer.deferredSentMessages[1:]
 			peer.mu.Unlock()
-			dm.callback(true, dm.seq)
+			err := dm.callback(true, dm.seq)
 			n.unblock()
 			n.waitBlocked()
 			peer.mu.Lock()
+			if err != nil {
```

Comment:
why not check `err` above?

Reply:
yes, that's a good point. I'll fix this in a subsequent PR.